charts/spire/charts/spire-server/values.yaml

```diff
@@ -188,12 +188,25 @@ upstreamAuthority:
     enabled: false
     rbac:
       create: true
-    issuer_name: "spire-ca"
+    # -- Defaults to the release name, override if CA is provided outside of the chart
+    issuer_name: ""
     issuer_kind: "Issuer"
     issuer_group: "cert-manager.io"
     # -- Specify to use a namespace other then the one the chart is installed into
     namespace: ""
     kube_config_file: ""
+
+    ca:
+      # -- Creates a Cert-manager CA
+      create: true
```

Comment:
Please put the default on false as in my previous comment on the orignal line. To keep experience the same as podmonitors etc.